cmd/oras/internal/option/packer.go

```diff
@@ -69,6 +71,35 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	}
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
+func (opts *Packer) Parse() error {
+	currentDir, err := os.Getwd()
+	var failedPaths []string
+	if err != nil {
+		return err
+	}
+	if !opts.PathValidationDisabled && len(opts.FileRefs) != 0 {
+		for _, path := range opts.FileRefs {
+			//Remove the type if specified in the path <file>[:<type>] format
+			lastIndex := strings.LastIndex(path, ":")
+			if lastIndex != -1 {
+				path = path[:lastIndex]
+			}
+			absPath, err := filepath.Abs(path)
```

Comment:
Better to use the `Parse` function provided in https://github.com/oras-project/oras/tree/main/cmd/oras/internal/fileref. to get `path` since is provides better cross-platform compability.

Reply:
Addressed and checked for absolute path